### DISCUSSION TITLE
jQuery included twice when using Yarn/Webpack

### DISCUSSION THREAD

**Discussion by mickel1138** [jquery/jquery-ui#2007] [Q&A]
Our package.json has jQuery as a dependency `3.5.1` and jQuery-UI has a it at `>1.8.0 & <4.0.0`
However when doing a Yarn install, two version of jQuery are installed. 3.5.1 and 3.6.0, which breaks jQuery-UI.

When I request jQuery to set at `3.6.0` or `^3.5.1` (3.5.1 and up) its works fine.
Before, a node_modules was added to jQuery-UI folder inside node_modules.

We have it working now, but just wanted to mention it for others who had similar issues, and comments in case we were doing something wrong?

**Comment by mgol** [comment]
That's more of a support question of how dependencies work with Yarn/npm so let me convert that to a discussion.

**Comment by tpatartmajeur** [comment]
@mickel1138 I have the same problem, how do you solve it?

Edit:
Ok i solved my issue

```
 "resolutions": {
    "jquery-ui/jquery": "^3.5.1"
  },
```
In the package.json

https://classic.yarnpkg.com/en/docs/selective-version-resolutions/

**Reply by mickel1138** [reply]
I gave the answer in the question, go basically make your jQuery accept newer versions.
